src/main/java/main/SortEmployee.java

```diff
@@ -0,0 +1,62 @@
+package main;
+
+import comparator.FirstNameSorted;
+import comparator.IdSorted;
+import comparator.LastNameSorted;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class SortEmployee {
+
+    /**
+     * Посортувати колекцію за допомогою Collections класу
+     * */
+    public Set<Employee> sortUsingCollections(Set<Employee> employeeSet){
+        List<Employee> sortedList = new ArrayList<>(employeeSet);
+        Collections.sort(sortedList);
+        Set<Employee> sortedSet = new LinkedHashSet<>(sortedList);
+            return sortedSet;
+    }
+
+    /**
+     * Посортувати колекцію за допомогою Collections класу та Comparator інтерфейсу
+     * */
+    public Set<Employee> sortUsingComparatorAndCollections(Set<Employee> employeeSet){
+        List<Employee> sortedList = new ArrayList<>(employeeSet);
+        Collections.sort(sortedList, new IdSorted().thenComparing(new FirstNameSorted()).thenComparing(new LastNameSorted()));
+        Set<Employee> employeeSortedSet = new LinkedHashSet<>(sortedList);
+        return employeeSortedSet;
+    }
+
+    /**
+     * Посортувати колекцію за допомогою Stream API
+     * */
+    public Set<Employee> sortUsingStream(Set<Employee> employeeSet){
+        Set<Employee> sortedSet =  employeeSet.stream()
+                .sorted()
+                .collect(Collectors.toCollection(LinkedHashSet::new));
+        return sortedSet;
```

Comment:
It's not a mistake but you can immediately return employeeSet.stream()
                .sorted()
                .collect(Collectors.toCollection(LinkedHashSet::new));